takoyaki/src/main/java/frc/robot/subsystems/CameraWithPID.java

```diff
@@ -0,0 +1,31 @@
+package frc.robot.subsystems;
+
+import frc.robot.subsystems.Camera;
+
+import org.photonvision.PhotonCamera;
+import org.photonvision.PhotonUtils;
+import org.photonvision.targeting.PhotonTrackedTarget;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.math.controller.PIDController;
+
+public class CameraWithPID {
```

Comment:
this also isn't a subsystem - it seems like the intent was linking the camera with a subsystem's PID controller? I think it would make more sense for us to link to a subsystem instead of directly to a `PIDController`, because not all PID controllers support camera input. to that end, then, this should probably be a command, i.e. you add your subsystem as a dependency and call into it once you have results from the `PhotonCamera`, etc.